syncer/sharding_group.go

```diff
@@ -416,7 +417,7 @@ func NewShardingGroupKeeper(tctx *tcontext.Context, cfg *config.SubTaskConfig) *
 		tctx:   tctx.WithLogger(tctx.L().WithFields(zap.String("component", "shard group keeper"))),
 	}
 	k.shardMetaSchema = cfg.MetaSchema
-	k.shardMetaTable = fmt.Sprintf(shardmeta.MetaTableFormat, cfg.Name)
```

Comment:
You can remove `shardmeta.MetaTableFormat` now.

Reply:
removed in https://github.com/pingcap/dm/pull/651/commits/fb7c8b3c69d821ba58a6297bfd19c123b0116b09